### PR TITLE
Adding Selection Sort in Java

### PR DESCRIPTION
**Pull request by Khushi2402**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ShehrozIrfan) [inline .vscode/settings.json]: This file is added by mistake?
If yes, I think we need to remove this file.

File: Java/Sorting Algorithm/SelectionSort.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+//Selection Sort in Java
```
Comment (ShehrozIrfan) [inline Java/Sorting Algorithm/SelectionSort.java:1]: Good

**Review by ShehrozIrfan — CHANGES_REQUESTED** [review]
Overall looks good.

Just need to remove the `vscode/settings.json`

File: .vscode/settings.json (hunk 1 above)
Comment (Khushi2402) [reply .vscode/settings.json]: Sorry, that was added by mistake. I removed it now. Could you please check it again?

**Review by ShehrozIrfan — APPROVED** [review]
Looks good to merge 🎉

**Comment by ShehrozIrfan** [comment]
Thanks for your contribution.

**Pull request merged**